tools/engine_tool/lib/src/commands/build_command.dart

```diff
@@ -24,20 +23,26 @@ final class BuildCommand extends CommandBase {
   }) {
     builds = runnableBuilds(environment, configs);
     // Add options here that are common to all queries.
-    argParser.addOption(
-      _configFlag,
-      abbr: 'c',
-      defaultsTo: 'host_debug',
-      help: 'Specify the build config to use',
-      allowed: <String>[
-        for (final GlobalBuild config in runnableBuilds(environment, configs))
-          config.name,
-      ],
-      allowedHelp: <String, String>{
-        for (final GlobalBuild config in runnableBuilds(environment, configs))
-          config.name: config.gn.join(' '),
-      },
-    );
+    argParser
+      ..addOption(
+        configFlag,
+        abbr: 'c',
+        defaultsTo: 'host_debug',
+        help: 'Specify the build config to use',
+        allowed: <String>[
+          for (final GlobalBuild config in runnableBuilds(environment, configs))
+            config.name,
+        ],
+        allowedHelp: <String, String>{
+          for (final GlobalBuild config in runnableBuilds(environment, configs))
+            config.name: config.gn.join(' '),
+        },
+      )
+      ..addFlag(
```

Comment:
I don't think we should overload 'build' to also run tests. We should instead add a separate 'test' command.